### DISCUSSION TITLE
Security / Secret scanning alerts should allow filtering by active

### DISCUSSION THREAD

**Discussion by cdobbyn** [community/community#64266] [Code Security]
### Select Topic Area

Product Feedback

### Body

This would be helpful in tracking down and eliminating the most critical alerts when you first turn this tool on.

**Comment by rainbow1016** [comment]
very good question!

**Comment by aashah** [comment]
Hey @cdobbyn, what do you mean by active?

1. Alerts with an "open" resolution?
2. this secret is still usable by provider XX (i.e. AWS) right now? From this feature: https://github.blog/changelog/2023-04-28-secret-scanning-now-supports-validation-checks-for-supported-partner-patterns/ ?

For 1, this it the default view with `is:open` in the filter box.

For 2, this is a very actively developed feature right now. I'm pretty sure this is on their roadmap, but I think the focus is on adding support for more partners right now.
